Makefile

```diff
@@ -4,3 +4,9 @@ upgrade:
 	pip-compile --upgrade -o requirements/testing.txt requirements/testing.in
 	pip-compile --upgrade -o requirements/travis.txt requirements/travis.in
 	pip-compile --upgrade -o requirements/aws.txt requirements/aws.in
+	bash util/post-pip-compile.sh \
+	    requirements/aws.txt \
+	    requirements/base.txt \
+	    requirements/pip-tools.txt \
+	    requirements/testing.txt \
+	    requirements/travis.txt \
```

Comment:
We learned that we can avoid this in most cases by setting the `CUSTOM_COMPILE_COMMAND` environment variable: https://github.com/edx/edx-platform/blob/master/Makefile#L68